packages/locale-kit/utils.ts

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-restricted-imports */
+import { promises as fs, readdirSync } from 'fs'
+import { difference, keys, uniq, without } from 'lodash'
+import * as path from 'path'
+import { getUsedKeys } from './ast'
+
+const SOURCE_PATH = path.join(__dirname, '..', 'maskbook', 'src')
+export const LOCALE_PATH = path.join(SOURCE_PATH, '_locales')
```

Comment:
Hi @septs now we got a new pattern in the dashboard and theme.

```
/pkg/locales/{zh,en,ja}.json
```

Reply:
OK